lib/galaxy_test/base/populators.py

```diff
@@ -2158,6 +2161,29 @@ def new_library(self, name):
         create_response = self.galaxy_interactor.post("libraries", data=data, admin=True, json=True)
         return create_response.json()
 
+    def _fetch_single_url_to_folder(self, file_type="auto", assert_ok=True):
```

Comment:
Not a big deal, but maybe rename the function since now is part of the public interface of the populator:
```suggestion
    def fetch_single_url_to_folder(self, file_type="auto", assert_ok=True):
```

Reply:
thanks, i've rebased that into the first commit